yarn-project/scripts/Earthfile

```diff
@@ -66,4 +66,29 @@ bench-comment:
     [ -f $BENCH_FOLDER/benchmark.json ] \
       && (cd /usr/src/yarn-project/scripts && AZTEC_BOT_COMMENTER_GITHUB_TOKEN=$AZTEC_BOT_COMMENTER_GITHUB_TOKEN yarn bench-comment) \
       || echo "No benchmark file found in $BENCH_FOLDER"
-  
\ No newline at end of file
+
+base-log-uploader:
+  # Install awscli on a fresh ubuntu, and copy the repo "scripts" folder, which we'll use to upload logs
+  # Note that we cannot do this LOCALLY because Earthly does not support using secrets locally
+  FROM ubuntu:noble
+  RUN apt update && \
+      apt install -y curl git jq unzip
+  RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m).zip" -o "awscliv2.zip" && \
+      unzip awscliv2.zip && \
+      ./aws/install --bin-dir /usr/local/bin --install-dir /usr/local/aws-cli --update && \
+      rm -rf aws awscliv2.zip
+  COPY ../../+scripts/scripts /usr/src/scripts
```

Comment:
Note that this image does not depend on yarn-project at all, so if we want to move it to somewhere reusable, it can very well live at `/scripts` instead of `/yarn-project/scripts`.

Reply:
I've moved this into the root earthfile (as there isn't one for `scripts`) [here](https://github.com/AztecProtocol/aztec-packages/pull/6424)